src/test/java/svc/data/citations/datasources/tyler/TylerCitationDataSourceIntegrationTest.java

```diff
@@ -13,8 +16,8 @@
 import org.springframework.test.context.junit4.SpringRunner;
 
 import svc.Application;
+import svc.models.Citation;
 
-//TODO
 @Ignore
```

Comment:
I didn't want to leave the test filled out in a way that would make someone's real citation public domain, so i took out the working information and left the test ignored so it could be used by developers as needed.